src/mono/wasm/runtime/managed-exports.ts

```diff
@@ -64,6 +65,7 @@ export function init_managed_exports(): void {
     };
     runtimeHelpers.javaScriptExports.release_js_owned_object_by_gc_handle = (gc_handle: GCHandle) => {
         mono_assert(gc_handle, "Must be valid gc_handle");
+        loaderHelpers.assert_runtime_running();
```

Comment:
It might make sense for cleanup methods like Dispose, Release to silently exit when the runtime has exited/aborted, so that calling them from a finally block doesn't trigger a double fault and hide whatever the actual error was.

Reply:
I have another PR which is dealing with proxy disposal, and this PR will make the other one more crisp about it.

Reply:
For workers, it's crucial to do cleanup right, not just abandon it.